app/models/test_track/fake/schema_loading.rb

```diff
@@ -0,0 +1,34 @@
+class TestTrack::Fake::SchemaLoading
+  attr_reader :split_hash
+
+  def initialize
+    @split_hash = _split_hash
+  end
```

Comment:
can we make this a public method called `split_hash` that memoizes the result of `_split_hash`? (or uses the `begin...end` style